app/assets/main/react_components/routes/lifestyle_footprints/footprint_form_container/components/footprint_form/FootprintForm.jsx

```diff
@@ -204,7 +209,8 @@ const FootprintForm = ({
             savedValue={getSavedAnswer()}
           />
         ) : (
-          result && (
+          result &&
+          (currentIndex - questionObjects.length < 2 ? (
```

Comment:
Has been added since ResultPage.jsx doesn't have responsibility for membership/payment page anymore, which has been moved to SignUpPage.jsx (as shown below)